### PR TITLE
Enable benchmark in CI

### PR DESCRIPTION
**Pull request by shashi**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -22,3 +22,7 @@ include("nf.jl")
 include("interface.jl")
 include("fuzz.jl")
 include("adjoints.jl")
+
+if haskey(ENV, "CI")
+    include("benchmark.jl")
```
Comment (ChrisRackauckas) [inline test/runtests.jl:27]: Maybe use the buildkite benchmark nodes?

Comment (shashi) [reply test/runtests.jl:27]: Example config I can copy-paste?

Comment (ChrisRackauckas) [reply test/runtests.jl:27]: https://github.com/SciML/DiffEqBase.jl/blob/master/.buildkite/pipeline.yml change and change the queue part to match https://github.com/SciML/SciMLBenchmarks.jl/blob/f44a419b90d20b381c2cb89cafbb40671c39aa12/.buildkite/pipeline.yml#L19-L20

**Comment by DilumAluthge** [comment]
@ChrisRackauckas @shashi You should see the Buildkite status now showing up on PRs in this repository.

**Comment by shashi** [comment]
New to this, how do i log in to see the error?

**Comment by DilumAluthge** [comment]
I just made the pipeline public. Can you see it now?

**Comment by shashi** [comment]
Thanks! I can see it.

```

2021-08-09 18:50:08 INFO   Searching for pipeline config...
2021-08-09 18:50:08 FATAL  Could not find a default pipeline configuration file. See `buildkite-agent pipeline upload --help` for more information.
🚨 Error: The command exited with status 1
```

Hmm what's the file we need?

**Comment by DilumAluthge** [comment]
My guess is it's looking for the default file, `.buildkite/pipeline.yml`.

**Comment by shashi** [comment]
Oh ok!

**Comment by shashi** [comment]
Some weird github outage/bug caused my commits to not show up here, but now I have it in master, things work! Thanks a lot @DilumAluthge

**Pull request merged**